File: app/src/main/java/com/subodh/rathaur/custom_textview_android_master/MainActivity.java
package com.subodh.rathaur.custom_textview_android_master;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.widget.TextView;

public class MainActivity extends AppCompatActivity {

    TextView tvFirst, tvSecond, tvThird, tvFourth;

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);
        tvFirst=findViewById(R.id.tv_first);
        tvSecond=findViewById(R.id.tv_second);
        tvThird=findViewById(R.id.tv_third);
        tvFourth=findViewById(R.id.tv_fourth);

        setFontType(tvFirst,tvSecond,tvThird,tvFourth);
    }

    private void setFontType(TextView tvFirst, TextView tvSecond, TextView tvThird, TextView tvFourth) {


    }
}

File: app/src/main/java/com/subodh/rathaur/customtextview/MainActivity.kt
package com.subodh.rathaur.customtextview

import android.os.Bundle
import android.widget.TextView
import androidx.appcompat.app.AppCompatActivity

class MainActivity : AppCompatActivity() {
    var tvFirst: TextView? = null
    var tvSecond: TextView? = null
    var tvThird: TextView? = null
    var tvFourth: TextView? = null
    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
        tvFirst = findViewById(R.id.tv_first)
        tvSecond = findViewById(R.id.tv_second)
        tvThird = findViewById(R.id.tv_third)
        tvFourth = findViewById(R.id.tv_fourth)
    }
}